src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.html

```diff
@@ -0,0 +1,53 @@
+<div
+    class="border rounded"
+    [style.max-width]="'780px'"
```

Comment:
Do we really need that here? If there is no special reason I would omit that so that instead of getting this: 
![image](https://user-images.githubusercontent.com/63286106/94397016-5555ab80-0163-11eb-8e03-80f0e2fd1b7c.png)
we could have that div simply adjust its with to the parent like this:
![image](https://user-images.githubusercontent.com/63286106/94397138-7a4a1e80-0163-11eb-89c6-67f797170376.png)

Reply:
I oriented this how it is done for GitHub. There the comment does not have the full width either and is around 780px width :) 
![image](https://user-images.githubusercontent.com/41108487/94404022-999a7900-016e-11eb-9fb3-52d526f04da6.png)

Reply:
oh okay I see :)